### PR TITLE
tools: fix mmlink Quartus 21.3 loopback reset

### PR DESCRIPTION
**Pull request by anandaravuri**
Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.h [hunk 1]
```diff
@@ -141,6 +141,7 @@ extern ST_DBG_IP_DESIGN_INFO g_std_dbg_ip_info;
 
 #define ST_DBG_IP_CONFIG_RESET_AND_LOOPBACK 0x20
 #define ST_DBG_IP_CONFIG_H2T_T2H_RESET_FIELD 0x1
+#define ST_DBG_IP_CONFIG_H2T_T2H_RESET_FIELD 0x1
```
Comment (tswhison) [inline tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.h:144]: Isn't this the same definition as on the previous line?  We should remove it for clarity.

File: tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.c [hunk 2]
```diff
@@ -245,11 +245,12 @@ int get_mgmt_rsp_data(MGMT_PACKET_HEADER *header, unsigned char **payload) {
 }
 
 void set_loopback_mode(int val) {
-    unsigned long rd = (unsigned long)IORD_64DIRECT(g_std_dbg_ip_info.ST_DBG_IP_CSR_BASE_ADDR, ST_DBG_IP_CONFIG_RESET_AND_LOOPBACK);
+
+    uint32_t rd = (uint32_t)IORD_32DIRECT(g_std_dbg_ip_info.ST_DBG_IP_CSR_BASE_ADDR, ST_DBG_IP_CONFIG_RESET_AND_LOOPBACK);
```
Comment (michael-adler) [inline tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.c:249]: How about some comments to explain what's going on?

File: tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.h (hunk 1 above)
Comment (anandaravuri) [reply tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.h]: fixed , removed duplicate  #define

File: tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.c (hunk 2 above)
Comment (anandaravuri) [reply tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.c:249]: fixed in remote-debug-for-intel-fpga
Adjust mmio rw 64 to 32:

https://github.com/altera-opensource/remote-debug-for-intel-fpga/commit/e04a93e100e34c2eeca5aa00d60d55413970f3ca

**Review by michael-adler — APPROVED** [review]
Now works on ADP with Quartus 21.3. Confirmed that reading signals with quartus_stpw is successful.

**Pull request merged**